Update deployment methods docs (#9608)
RasaHQ/rasa: docs/docs/deploy/deploy-rasa.mdx

---
id: deploy-rasa
sidebar_label: Deploy Rasa Assistant
title: Deploy Rasa Assistant

> description: How to deploy your Rasa Assistant with Kubernetes/Openshift

Reviewer: ```suggestion
description: The first of three steps to deploy your Rasa assistant
```

> abstract: This page explains how to deploy Rasa Assistant by using Helm.

Reviewer: ```suggestion
abstract: This page explains how to deploy Rasa Open Source using Helm. The first of three steps in deploying your Rasa assistant.
```
